src/vstart.sh

```diff
@@ -66,7 +67,10 @@ fi
 
 [ -z "$PYBIND" ] && PYBIND=./pybind
 
-export PYTHONPATH=$PYBIND:$CEPH_LIB/cython_modules/lib.${CEPH_PY_VERSION_MAJOR}:$PYTHONPATH
+[-n "$CEPH_PYTHON_COMMON" ] && CEPH_PYTHON_COMMON="$CEPH_PYTHON_COMMON:"
```

Comment:
please add a space before `-n`. and under what circumstances, we will have an empty `$CEPH_PYTHON_COMMON` ?

Reply:
If `$CEPH_BUILD_ROOT` is set explicitly, then `make install` should install python packages properly so we don't need to modify `PYTHONPATH`.

See 

https://github.com/ceph/ceph/blob/2a733cd4ea1197ebc441a64ed1d72136397f77df/src/vstart.sh#L51

Reply:
fixed